apps/condo/domains/property/components/panels/Builder/forms/SectionForm.tsx

```diff
@@ -0,0 +1,266 @@
+/** @jsx jsx */
```

Comment:
These components are very similar to `CreateParkingForm` and `EditParkigForm`.
Is there any way we can reuse them?